src/main/java/Task.java

```diff
@@ -0,0 +1,18 @@
+public class Task {
+    protected String item;
+    protected boolean complete;
```

Comment:
Boolean variables should sound like a claim that can be true or false. isComplete may be more appropriate here.